src/dotnet-svcutil/lib/src/AppInsightsTelemetryClient.cs

```diff
@@ -92,14 +92,8 @@ public static async Task<AppInsightsTelemetryClient> GetInstanceAsync(Cancellati
                             }
 
                             TelemetryConfiguration config;
-                            try
-                            {
-                                config = TelemetryConfiguration.Active;
```

Comment:
@HongGit I was removing this try block because after upgrading referenced package _Microsoft.ApplicationInsights_  from 2.2.0 to 2.20.0, build throws:

>  error CS0618: 'TelemetryConfiguration.Active' is obsolete: 'We do not recommend using TelemetryConfiguration.Active on .NET Core. See [https://github.com/microsoft/ApplicationInsights-dotnet/issues/1152](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FApplicationInsights-dotnet%2Fissues%2F1152&data=05%7C01%7Cv-carwan%40microsoft.com%7C09c7f6d19865496493be08da39b68fe1%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637885753526158443%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Zl%2F5Jbq%2BJ8VOnzBH8mB3i%2BjaxLetO6L7xunHzIu2ahs%3D&reserved=0) for more details'